esp/esdllib/esdl_def.cpp

```diff
@@ -1279,7 +1279,7 @@ class EsdlDefServiceArrayIterator : public CInterface, implements IEsdlDefServic
 {
 private:
     ESDLServiceArray &   array;
-    aindex_t             cur;
+    aindex_t             cur = aindex_t(-1);
```

Comment:
This is ok but it may even be better to use the `NotFound` const from jarray.cpp

Reply:
Change made.